backend/api/Startup.cs

```diff
@@ -129,6 +139,17 @@ public void ConfigureServices(IServiceCollection services)
             // Controller is scoped because a new instance should be initialized for each request
             services.AddScoped(typeof(ServiceNowController), typeof(ServiceNowController));
             #endregion
+
+            //WBS API Integration
+            #region WBS API Integration
+
+            // Service is of singleton type because it should be the same for all requests
+            // TODO: Verify if this should be singleton
+            services.AddScoped(typeof(WbsService), typeof(WbsService));
```

Comment:
This service is scoped, so this comment doesn't make sense

Reply:
Fixed